resources/aws/routetable.go

```diff
@@ -112,3 +112,43 @@ func (r *RouteTable) Delete() error {
 func (r RouteTable) ID() string {
 	return r.id
 }
+
+func (r RouteTable) getInternetGateway() (string, error) {
+	resp, err := r.Client.DescribeInternetGateways(&ec2.DescribeInternetGatewaysInput{
+		Filters: []*ec2.Filter{
+			&ec2.Filter{
+				Name: aws.String("attachment.vpc-id"),
```

Comment:
What is this magic string about? Some comments for the function would probably be good. Looking at the code it tries to lookup some internet gateway ID for the VPC the route table is configured for. I don't know what the internet gateway ID is.